src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int oneHundred = 100;

```

Comment:
```suggestion
    private static final int QUANTITY_LIMIT = 100;
```
don't name constants with their value - use constant name to tell people who will read your code about the role it serves in the application,